### ISSUE TITLE
The indexes always are zero (v0.26.1 only)

### ISSUE THREAD

**Issue by garytan0722** [asticode/go-astisub#98]
Hi,
In version `0.26.1`, the indexes for .srt or .vtt files consistently register as zero, whereas in version `0.26.0`, the behavior is normal.
Here is simplified code and [subtitle files](https://github.com/asticode/go-astisub/files/13855749/subtitle.zip):  
```
s1, _ := astisub.OpenFile("test.srt")

for _, v := range s1.Items {
	fmt.Printf("Index: %d\n", v.Index)
}
```

**Comment by asticode** [comment]
I've pushed a fix on master and I've created a `v0.26.2` tag, could you confirm it's working properly?

**Comment by garytan0722** [comment]
It works, thank for your help.
